app/views/surveys/_survey_definitions.html.erb

```diff
@@ -0,0 +1,54 @@
+<script type="application/json" data-survey-default>
+  {
+    "url": "https://www.surveymonkey.com/s/2MRDLTW?c={{currentPath}}",
+    "identifier": "user_satisfaction_survey",
+    "frequency": 50,
+    "surveyType": "url",
+  }
+</script>
```

Comment:
Quite wary of this, does a chunk of JSON on the page open us up to any potential security or injection problems?